admin/src/js/controllers/multiple-user.js

```diff
@@ -0,0 +1,151 @@
+angular.module('controllers').controller('MultipleUserCtrl', function(
```

Comment:
Please add unit test for the scope functions since those are public

Reply:
Do you have any specific tests you think are necessary or simply making sure the scope functions are called and have the correct values is sufficient?

Reply:
Yes, it's not that much actually:
- $scope.onCancel is cleaning, setting variables correctly and closing modal
- $scope.processUpload is setting the scope correctly based on a mock log, the outputFileUrl is correct and it's catching errors 
- $scope.backToAppManagement navigates and cleans correctly 
- $scope.showFinishSummary and  $scope.showDisplayUploadConfirm cleans and set just the necessary